src/core/util.ts

```diff
@@ -20,10 +20,8 @@ export const pickedObjectEquals = (picked: any, element: any) =>
   !!picked &&
   (picked === element ||
     (!!picked.id && picked.id === element) ||
-    (!!picked.primitive &&
-      (picked.primitive === element ||
-        (!!picked.primitive.equals && picked.primitive.equals(element)) ||
-        polylineEquals(picked.primitive, element))));
+    (!!picked.equals && picked.equals(element)) ||
+    polylineEquals(picked, element));
```

Comment:
```suggestion
    (picked.equals?.(element)) || polylineEquals(picked, element));
```

Reply:
going ahead and doing that, i only didn't do that because that is what is was before